cmd/bnscli/cmd_gov.go

```diff
@@ -257,103 +257,31 @@ func inOneHour() time.Time {
 	return time.Now().Add(time.Hour)
 }
 
-/*
-#!/bin/bash
```

Comment:
This comment is a great time saver when transaction declaration is changing. Did you intend to move it to a separate file and forgot to add the file?

Reply:
no, this came without a comment so I assumed it is dead code. Can you elaborate what it does and why we should keep it (in a separate file probably?), please

Reply:
Maybe it's time to make it a separate file.

This is a script for generating Go code https://github.com/iov-one/weave/blob/master/cmd/bnscli/cmd_gov.go#L51-L60
It saves a lot of time (and bugs) creating type casting and conversion for messages used by `bnsd` transactions.

Reply:
extracted to a new file for doc not execution